index.d.ts

```diff
@@ -0,0 +1,2 @@
+import freeEmailDomains from './domains.json' assert { type: 'json' }
+export default freeEmailDomains
```

Comment:
I used ESM-style `import` because `require()` was resulting in type of `any`, whereas `import` with the `type: 'json'` import assertion results in `string[]`:

<img width="288" alt="Screenshot 2022-11-15 at 14 19 59" src="https://user-images.githubusercontent.com/1935696/201930224-44188f21-5f75-44f6-9176-b9d2cab0706b.png">